src/view.rs

```diff
@@ -10,12 +12,12 @@ use crate::{utils::layout_on_viewport, ListState};
 /// A struct representing a list view.
 /// The widget displays a scrollable list of items.
 #[allow(clippy::module_name_repetitions)]
-pub struct ListView<'a, T> {
+pub struct ListView<'block, 'render, T> {
```

Comment:
Can we get by with just one lifetime here? 

This way we don't have to juggle 2 lifetimes. And all examples will compile without changes which is a good indication that most users don't need changes.

Reply:
Yes we totally can, and that makes way more sense anyways since there can't be many cases where these need to be separate lifetimes.